### ISSUE TITLE
Failure message not helpful

### ISSUE THREAD

**Issue by pitrou** [conda/conda-build#736]
When applying a patch fails, conda-build doesn't output the error message of applying the patch, which makes things difficult to debug:

``````
subprocess.CalledProcessError: Command '['c:\\var\\jenkins\\workspace\\LLVMDev\\label\\7x64b\\miniconda\\Library\\bin\\patch.exe', '-p0', '-i', 'c:\\var\\jenkins\\workspace\\LLVMDev\\label\\7x64b\\source\\conda-recipes\\llvmdev-jenkins\\../llvm-lto-static.patch_unix']' returned non-zero exit status 1```
``````

**Comment by wojdyr** [comment]
this problem is gone in the just released v1.19.0

**Comment by jakirkham** [comment]
@pitrou, can you verify this is fixed in 1.19.0?

**Comment by jakirkham** [comment]
Should be closed I think.
